app/views/provider/admin/backend_apis/metrics/index.html.slim

```diff
@@ -29,7 +29,7 @@ section.Section.u-toggleableBySettingsBox data-state="open"
 
     tbody
       - if @backend_api.method_metrics.any?
-        - @backend_api.method_metrics.each do |method|
+        - @backend_api.method_metrics.includes(:proxy_rules, :parent).each do |method|
```

Comment:
PRobably safer (although maybe there is no need 😄 :

```suggestion
        - @backend_api.method_metrics.includes(:proxy_rules, :parent).find_each do |method|
```

Reply:
I prefer not to touch it. It's a READ operation for the UI only. If it's too heavy for the database, then so it is for the UI.

Reply:
I am not as worried for the DB as for the memory it would need at once 😄 
However, this is a very good point: Then it is also too heavy for the UI 👍